tools/pipelines/build-docs.yml

```diff
@@ -80,228 +72,25 @@ variables:
-          displayName: 'Publish Guardian Artifacts - All Tools'
-          condition: succeededOrFailed()
-          inputs:
-            ArtifactType: M365
-
-        - task: AssetRetention@5
-          displayName: Guardian Asset Retention
-          condition: and(succeeded(), eq(variables.shouldRetainGuardianAssets, true))
-          inputs:
-            ArrowServiceConnection: 'ff-internal-arrow-sc'
-            AssetGroupName: 'fluidframework_$(System.TeamProject)_$(Build.DefinitionName)'
-            AssetNumber: '$(Build.BuildId)'
-            IsShipped: false # based on value of arrow.releasedtoproduction variable
-            DropsToRetain: 'CodeAnalysisLogs'
-
-        - task: securedevelopmentteam.vss-secure-development-tools.build-task-postanalysis.PostAnalysis@2
-          displayName: 'Guardian Break'
-          condition: succeededOrFailed()
-          continueOnError: false
-          inputs:
-            GdnBreakPolicyMinSev: Warning
-            GdnBreakAllTools: true
-            GdnBreakBaselineFiles: '$(Build.SourcesDirectory)/docs/.gdnbaselines'
-            GdnBreakGdnToolESLint: true
-            GdnBreakGdnToolESLintSeverity: Warning
-            GdnBreakPolicy: M365
-            GdnBreakOutputBaselineFile: '$(Build.ArtifactStagingDirectory)/'
-
-        - task: PublishPipelineArtifact@1
-          displayName: 'Publish Baselines'
-          condition: eq('${{ parameters.publishGuardianBaselines }}', 'true')
-          inputs:
-            targetPath: '$(Build.ArtifactStagingDirectory)/.gdnbaselines'
-            artifactName: .gdn
-# END Secure development tasks
-
-- stage: link_check
-  displayName: 'Website Link Check'
-  dependsOn: [] # run in parallel
-  pool: Large
-  jobs:
-    - job: link_check
-      displayName: 'Website Link Check'
-      continueOnError: true
-      steps:
-        - checkout: self
-          submodules: false
-          clean: true
-
-        - template: templates/include-use-node-version.yml
-
-        - template: templates/include-install-pnpm.yml
-          parameters:
-            buildDirectory: $(Build.SourcesDirectory)/docs
-
-        - task: Bash@3
-          displayName: Install dependencies
-          inputs:
-            targetType: 'inline'
-            workingDirectory: $(Build.SourcesDirectory)/docs
-            script: |
-              set -eu -o pipefail
-              pnpm i --frozen-lockfile
-
-        - task: Npm@1
-          displayName: npm run build
-          inputs:
-            command: 'custom'
-            workingDir: $(Build.SourcesDirectory)/docs
-            customCommand: 'run ci:build'
-
-        - task: Npm@1
-          displayName: npm run ci:linkcheck
-          inputs:
-            command: 'custom'
-            workingDir: $(Build.SourcesDirectory)/docs
-            customCommand: 'run ci:linkcheck'
-
 - stage: check_branch_version
```

Comment:
(Sorry, I'm a stickler for docs 😋)

Can we document each of the stages here? The last one, in particular, is interesting since it's triggering another pipeline to run (if I'm understanding correctly).